packages/terra-time-input/src/_AccessibleInput.jsx

```diff
@@ -66,9 +66,15 @@ function AccessibleInput(props) {
   const uuid = uuidv4();
   const labelId = `${uuid}-label`;
   const descriptionId = `${uuid}-description`;
+  let labelString;
+
+  if (inputProps && inputProps.fieldLabel) {
+    labelString = inputProps.fieldLabel;
+  }
+
```

Comment:
These code changes will not work if the consumer passes the prop name as label or anything else instead of fieldLabel. How will the consumer know that they have to pass fieldLabel always to the inputAttributes?

Reply:
This is added because of there is no field component in dateTimePicker and to handle a reading of single label in both the component require to use it through fieldLabel .

Reply:
Have updated the description regarding timeInputAttribute for this scenario.